spec/features/users/login_spec.rb

```diff
@@ -110,9 +112,10 @@
       click_button "Submit"
 
       visit '/merchant'
+
       expect(page).to have_content("The page you were looking for doesn't exist.")
 
-      visit admin_path
+      visit '/admin'
```

Comment:
I like admin_path better here.

Reply:
Sorry, left over from when I was debugging. Just updated in the Github file.